odoo_module_migrate/migration_scripts/migrate_120_130.py

```diff
@@ -2,6 +2,21 @@
 # @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
+_TEXT_WARNINGS = {
+    ".xml": {
+        r'active.*widget="toggle_button"|widget="toggle_button".*active|'
+        r'widget="boolean_button".*active|active.*widget="boolean_button"':
+        'You can remove actions archive/unarchive'
+    },
+    ".py": {
+        r".*oldname":
+            "oldname is not supported yet. Create a migration script",
+        r"digits=[^, )]*get_precision":
```

Comment:
Thank you @flachica For this enhancement. `Text Warnings` functionality could be useful enhancement. BUT IMO, ome of the proposed warning should be fixed by the migration script. `api.returms`should be automatically removed and the call to `get_precision` replaced by string argument only.

Reply:
I remove all text warnings. In future PR this can be included